modules/xfeatures2d/src/teblid.p256.hpp

```diff
@@ -0,0 +1,102 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Author: Iago Suarez <iagosuarz@gmail.com>
+
+// Implementation of the article:
+//     Iago Suarez, Jose M. Buenaposada, and Luis Baumela.
+//     Revisiting Binary Local Image Description for Resource Limited Devices.
+//     IEEE Robotics and Automation Letters, vol. 6, no. 4, pp. 8317-8324, Oct. 2021.
+
+// ABWLParams: x1, y1, x2, y2, boxRadius, th
+
+// Pre-trained parameters of TEBLID-256 trained in Liberty data set. 10K triplets are sampled per iteration. Each triplet
+// contains an anchor patch, a positive and a negative, selected as the hardest among 256 random negatives.
+static const std::vector<ABWLParamsFloatTh> teblid_wl_params_256 = {
```

Comment:
> std::vector<ABWLParamsFloatTh>

Need to check generated binary code of this initializer. Array should be fine, but `std::vector` - doesn't.

Could you please use something like this:

```
static const ABWLParamsFloatTh teblid_wl_params_256_[] = {...};
static const std::vector<ABWLParamsFloatTh> teblid_wl_params_256(teblid_wl_params_256_, teblid_wl_params_256 + sizeof(teblid_wl_params_256_) / sizeof(teblid_wl_params_256_[0]));
```

Reply:
Sorry for late reply due to vacations and thanks again for your comments.

I have fixed it in my last commit.